en, zh: update tkctl debug usage (#898)
pingcap/docs-tidb-operator: en/use-tkctl.md

### tkctl debug [pod_name]

> This command is used to diagnose the Pods in a TiDB cluster. It launches a debug launcher Pod and then the Pod launches a container with the specified docker image on the host that holds the target Pod. The container has the necessary troubleshooting tools installed and shares the namespace with the container in the target Pod, so you can seamlessly diagnose the target container by using various tools in the debug container.

Reviewer: ```suggestion
This command is used to diagnose the Pods in a TiDB cluster. It launches a debug launcher Pod which then starts a debug container using specified docker image on the same host of the target Pod. The container has necessary debug tools and shares the same namespace with the container in the target Pod, so you can diagnose the target container by using debug tools unaware of the debug container.
```

Reviewer: rest LGTM

Author: ```suggestion
This command is used to diagnose the Pods in a TiDB cluster. It launches a debug launcher Pod which then starts a debug container using the specified docker image on the same host of the target Pod. The container has necessary troubleshooting tools and shares the same namespaces with the container in the target Pod, so you can diagnose the target container by using various tools in the debug container.
```

| Flag | Abbreviation | Description |
| ----- | --------- | ----------- |
| --image |    | Specify the docker image used by the debug container; default to `pingcap/tidb-debug:lastest` |

> | --launcher-image         |           | Specify the docker image for the debug launcher pod which is responsible to launch the debug container; default to `pingcap/debug-launcher:latest`                |

Reviewer 2: ```suggestion
| --launcher-image         |           | Specify the docker image for the debug launcher pod which is responsible for launching the debug container; default to `pingcap/debug-launcher:latest`                |
```
